.github/scripts/instr-count-benchmark/create-nightly-env.sh

```diff
@@ -23,16 +24,22 @@ conda install -y valgrind -c conda-forge
 
 # Check if nightly builds are available
 NIGHTLIES=$(python torchbenchmark/util/torch_nightly.py --packages torch)
+echo "${NIGHTLIES}"
 
 # If failed, the script will generate empty result
-if [ -z $NIGHTLIES ]; then
+if [ -z "${NIGHTLIES}" ]; then
     echo "Torch nightly build failed. Cancel the workflow."
     exit 1
 fi
 
+TORCH_VERSION_STR=$(python torchbenchmark/util/torch_nightly.py --packages torch --pip_str)
+
 # Install PyTorch nightly from pip
-pip install --pre torch \
+pip install --pre torch=="${TORCH_VERSION_STR}" \
```

Comment:
May I ask why you are installing pytorch with the version number get from torch_nightly.py?